package.json

```diff
@@ -54,7 +55,7 @@
     "test:fast": "npm run build:fast && npm run test",
     "ci": "npm run test && catch-uncommitted",
     "watch": "gulp watch",
-    "prettify": "prettier --write \"{lib,tests,automation,typings}/**/*.ts\" --config ./node_modules/resin-lint/config/.prettierrc",
+    "prettify": "prettier --write \"{lib,tests,automation,typings}/**/*.[tj]s\" --config ./node_modules/resin-lint/config/.prettierrc",
```

Comment:
These expansions can fail when ran on certain shells, I've been bitten by this myself

Reply:
In this case, it is `prettier` itself that interprets the glob, not a shell: https://prettier.io/docs/en/cli.html

By the way, I've also tested it on Windows, with `cmd.exe`, which is fairly different from sh or bash...